### PR TITLE
Feature migrating to v5

### PR DESCRIPTION
**Pull request by Melissa-Martinez-XL**
## Ticket

* [Migrating Gatsby from v4 to v5](https://www.notion.so/xmartlabs/Migrating-Gatsby-from-v4-to-v5-87469d8269754844927031ade63b0778?pvs=4)

## Type of change

* [ ] Fix
* [ ] Story
* [X] Chore

## Description of the change

I migrated gatsby from v3/v4 to v5, and I also changed the correspondent plugins.

## Screenshot/Execution

Relevant screenshots or steps to execute a task

## Related PRs
Links to the related PRs

### CODE REVIEW

**Comment by Melissa-Martinez-XL** [comment]
@gpicart @lauta2912 You should ignore the blog files, they only have some syntax changes.

File: blogs/2015-07-09-Android-logging-with-Crashlytics-and-Timber.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ After all, why do we have to do this? This is boilerplate. You have to add it to
 [Crashlytics](https://get.fabric.io/crashlytics), part of [Fabric](https://get.fabric.io/), is there for you. Crashes and logged exceptions will be notified to you via email if you decide to use it. More information about the issues is available, like the whole stack trace, Android versions that caused the issue, device models and even more details you want to add.
 
 To setup Crashlytics in your app you have to add this to your `build.gradle`:
-
+```
```
Comment (lauta2912) [inline blogs/2015-07-09-Android-logging-with-Crashlytics-and-Timber.md:39]: If you add this one here, you should add one to close the snippet

File: blogs/2016-03-07-ci-with-xcode-server-and-fastlane.md [hunk 2]
```diff
@@ -481,10 +481,8 @@ $ fastlane before_integration_staging
 ```
 
 On the output log appears next messages:
-
-    security: SecKeychainAddInternetPassword <NULL>: User interaction is not allowed.
+    security: SecKeychainAddInternetPassword `<NULL/>`: User interaction is not allowed.
```
Comment (lauta2912) [inline blogs/2016-03-07-ci-with-xcode-server-and-fastlane.md:484]: The change from <NULL> to <NULL/> is not acceptable. The author meant to write it without the `/`

File: blogs/2016-03-07-ci-with-xcode-server-and-fastlane.md [hunk 3]
```diff
@@ -512,9 +510,8 @@ $ ln -s `which pod` /Applications/Xcode.app/Contents/Developer/usr/bin/pod
 #### Fastlane - Sigh & Gym cannot access to keychain
 
 That's all, they cannot access to keychain. Seeing this message (or similar) when running `gym`or `sigh` is the symptom:
-
-    security: SecKeychainAddInternetPassword <NULL>: User interaction is not allowed.
-
+    security: SecKeychainAddInternetPassword `<NULL/>`: User interaction is not allowed.
```
Comment (lauta2912) [inline blogs/2016-03-07-ci-with-xcode-server-and-fastlane.md:513]: Same here

File: blogs/2016-10-21-The-First-Eureka-Webinar.md [hunk 4]
```diff
@@ -30,6 +30,6 @@ On the agenda:
 - Eureka case studies
 - Q&A
 
-Feel free to <getintouch@xmartlabs.com> if you have any questions or would like to make a special request for the webinar.
+Feel free to getintouch@xmartlabs.com if you have any questions or would like to make a special request for the webinar.
```
Comment (lauta2912) [inline blogs/2016-10-21-The-First-Eureka-Webinar.md:33]: Same here, I believe? Maybe it has the same behavior. can you check?

File: blogs/2020-03-30-ios-continuous-integration-delivery-using-fastlane-and-bitrise.md [hunk 5]
```diff
@@ -111,7 +111,7 @@ Here is the entire workflow already created in Bitrise.
 At this point, the `test` lane will be executed whenever a pull request is created/updated. Cool right? Bitrise integrates with GitHub/Bitbucket and many other git solutions to indicate if everything goes well or not.
 
 
-Another automated task we strongly use is the release of new versions. Basically we send a new build to TestFlight whenever a new tag (named <version_*>) is created in . As we said Bitrise provides us with push, pull request and tag triggers, this time we are going to use a tag trigger.
+Another automated task we strongly use is the release of new versions. Basically we send a new build to TestFlight whenever a new tag (named `<version_*/>`) is created in . As we said Bitrise provides us with push, pull request and tag triggers, this time we are going to use a tag trigger.
```
Comment (lauta2912) [inline blogs/2020-03-30-ios-continuous-integration-delivery-using-fastlane-and-bitrise.md:114]: Same comment about the slash

File: blogs/2020-05-25-android-nav-component-part-1.md [hunk 6]
```diff
@@ -74,8 +74,8 @@ Although I didn't get to use it a lot, I found it useful and easy to use wheneve
 
 ### Jetpack & Android Architecture Components
 
-<div style="display: inline-block;">
-  <div style="float: left; width: 20%; margin-right: 1em;">
+<div style={{ display:'inline-block'}}>
+  <div style={{ flost:'left', width: '20%', marginRight: '1em'}}>
```
Comment (lauta2912) [inline blogs/2020-05-25-android-nav-component-part-1.md:78]: typo here

File: blogs/2020-10-02-react-native-line.md [hunk 7]
```diff
@@ -15,7 +15,8 @@ When one of our projects called for an integration of sign in with [Line](https:
 
 <img src="/images/react-native-line/example1.png" alt="Example app main" width="300" />
 <img src="/images/react-native-line/example2.png" alt="Example app permissions" width="300" />
-<p style="text-align: center; color: gray; font-size: 14px; font-style: italic;">(Example app with <code>react-native-line</code> integration)</p>
+<p style={{ textAlign:'center', color: 'gray', fontSize: '14px', fontStyle: 'italic'}}
+>(Example app with <code>react-native-line</code> integration)</p>
```
Comment (lauta2912) [inline blogs/2020-10-02-react-native-line.md:19]: does this line look the same way as before?

File: blogs/2021-07-30-Introducing-pagertabstrip-swiftui-pager-view.md [hunk 8]
```diff
@@ -10,7 +10,7 @@ twitter_image: /images/pager-swiftui/Twitter-1.png
 permalink: /introducing-pagertabstripview-pagerView-in-swiftui/
 ---
 
-<p style="text-align:left;width:100%">
+<p style={{ textAlign:'left', widht: '100%'}}>
```
Comment (lauta2912) [inline blogs/2021-07-30-Introducing-pagertabstrip-swiftui-pager-view.md:13]: typo here

File: gatsby-node.js [hunk 9]
```diff
@@ -28,13 +28,27 @@ exports.createResolvers = ({ createResolvers }) => {
...
       },
     },
   }
   createResolvers(resolvers)
 };
 
+const readingTime = require(`reading-time`)
+
+exports.onCreateNode = ({ node, actions }) => {
+  const { createNodeField } = actions
```
Comment (lauta2912) [inline gatsby-node.js:42]:
```suggestion
  const { createNodeField } = actions;
```

**Review by lauta2912 — CHANGES_REQUESTED** [review]

File: blogs/2015-09-29-Introducing-Eureka-iOS-form-library-written-in-pure-Swift.md [hunk 10]
```diff
@@ -113,9 +113,9 @@ Eureka uses Swift's [type safety] to help avoiding mistakes while developing. Ea
 
 For example in the code example shown above `$0` is an instance of `SegmentedRow<Emoji>` and its `value` property type is `Emoji` whereas NameRow's `value` property type is `String`. Similarly `SegmentedRow<Emoji>`'s `options` array property only stores `Emoji` instances and `options` is not available in `NameRow`.
 The same applies to any chainable method we invoke to set up a row, for example `cell` and `row` parameters of `cellUpdate` and `cellSetup` closure are also strongly typed.
-
+```
```
[the remark this answers is not in the document]
Comment (Melissa-Martinez-XL) [reply blogs/2015-09-29-Introducing-Eureka-iOS-form-library-written-in-pure-Swift.md]: Done

File: blogs/2015-07-09-Android-logging-with-Crashlytics-and-Timber.md (hunk 1 above)
Comment (Melissa-Martinez-XL) [reply blogs/2015-07-09-Android-logging-with-Crashlytics-and-Timber.md:39]: Done

File: blogs/2016-03-07-ci-with-xcode-server-and-fastlane.md (hunk 2 above)
Comment (Melissa-Martinez-XL) [reply blogs/2016-03-07-ci-with-xcode-server-and-fastlane.md:484]: Done

File: blogs/2016-03-07-ci-with-xcode-server-and-fastlane.md (hunk 3 above)
Comment (Melissa-Martinez-XL) [reply blogs/2016-03-07-ci-with-xcode-server-and-fastlane.md:513]: Done

File: blogs/2016-10-21-The-First-Eureka-Webinar.md (hunk 4 above)
Comment (Melissa-Martinez-XL) [reply blogs/2016-10-21-The-First-Eureka-Webinar.md:33]: Done

File: blogs/2017-05-01-Introducing-to-Ahoy.md [hunk 11]
```diff
@@ -97,4 +97,5 @@ We hope that Ahoy will be as useful for you as it was for us and that you create
 
 
 Need other quick but powerful solutions?
-Visit us on [GitHub](https://github.com/xmartlabs), we’re one of the top 10 GitHub Swift developers :), or drop us a line at <hi@xmartlabs.com>.
+Visit us on [GitHub](https://github.com/xmartlabs), we’re one of the top 10 GitHub Swift developers :), or drop us a line at hi@xmartlabs.com.
```
[the remark this answers is not in the document]
Comment (Melissa-Martinez-XL) [reply blogs/2017-05-01-Introducing-to-Ahoy.md:100]: Done

File: blogs/2017-09-25-iOS11.md [hunk 12]
```diff
@@ -14,36 +14,36 @@ This new release includes new features and technologies such as... don't be so a
...
   <img src='https://thecryptosphere.files.wordpress.com/2016/01/mind-blown.gif' width= '200px' alt='Blow mind'/>
 </p>
 
 - Live photos have some new tricks that you'll also love. If you turned off this feature, it's time to turn it back on! With iOS 11 this little clip can be edited and make it loop, bounce back, or create a long exposure video.
 
-<div style= "display:flex; flex-direction:row; height: 350px">
-  <img src='https://cnet1.cbsistatic.com/img/3Yc2TbMsUsY-EGFaV6S6QNqUrko=/fit-in/x/2017/06/22/fa918c6c-2357-46b9-9ce0-4a2a03fbc7f2/loop-live-photos-ios-11.gif' width= '300px' alt='loop' style="flex:1"/>
-  <img src='https://cnet3.cbsistatic.com/img/N_CZjIB5L3yVbs-v3lsJyKe9vhA=/fit-in/x/2017/06/22/a1847000-3008-46d0-a1ba-90148e4a5ed2/bounce-live-photos-ios-11.gif' width= '300px' alt='bounce' style="flex:1"/>
-  <img src='https://cnet2.cbsistatic.com/img/z-34DGqjuapzGLUv6t23qf04X_4=/fit-in/x/2017/06/22/c2a0b35e-8363-427e-8861-eaeefaff968c/long-exposure-live-photos-ios-11.jpg' width= '300px' alt='long exposure' style="flex:1"/>
+<div>
```
[the remark this answers is not in the document]
Comment (Melissa-Martinez-XL) [reply blogs/2017-09-25-iOS11.md:43]: The links to the pictures are not working, Do I have to delete them or no?

File: blogs/2020-03-30-ios-continuous-integration-delivery-using-fastlane-and-bitrise.md (hunk 5 above)
Comment (Melissa-Martinez-XL) [reply blogs/2020-03-30-ios-continuous-integration-delivery-using-fastlane-and-bitrise.md:114]: Done

File: blogs/2020-05-25-android-nav-component-part-1.md (hunk 6 above)
Comment (Melissa-Martinez-XL) [reply blogs/2020-05-25-android-nav-component-part-1.md:78]: Done

File: blogs/2020-12-31-recap2020.md [hunk 13]
```diff
@@ -838,7 +838,7 @@ If you often receive audio texts at wrong time... the app is for you! Download i
             }}
           />
         </div>
-      </div>{" "}
```
[the remark this answers is not in the document]
Comment (Melissa-Martinez-XL) [reply blogs/2020-12-31-recap2020.md:841]: Yes, in the deploy looks okay

File: blogs/2021-07-30-Introducing-pagertabstrip-swiftui-pager-view.md (hunk 8 above)
Comment (Melissa-Martinez-XL) [reply blogs/2021-07-30-Introducing-pagertabstrip-swiftui-pager-view.md:13]: Done

File: blogs/2022-02-24-intro-to-recommendations-engine.md [hunk 14]
```diff
@@ -34,13 +34,13 @@ The main reason to adopt a recommendation engine is simple: **increasing your re
 
 
 
-<!-- ![Don't take our word for it, trust the numbers.](/images/recommendation-engines/REC-SYSTEMS.png) -->
```
[the remark this answers is not in the document]
Comment (Melissa-Martinez-XL) [reply blogs/2022-02-24-intro-to-recommendations-engine.md:37]: The image is showing
Captura desde 2023-05-16 14-53-52

File: gatsby-node.js (hunk 9 above)
Comment (Melissa-Martinez-XL) [reply gatsby-node.js]: Done

**Review by gpicart — APPROVED** [review]

**Review by gpicart — APPROVED** [review]

**Review by gpicart — APPROVED** [review]

**Pull request merged**